app/js/components/labOrderEntry/LabEntryForm.jsx

```diff
@@ -14,46 +18,54 @@ export class LabEntryForm extends React.Component {
     categoryId: 1,
     defaultTests: [],
     selectedTests: [],
-    selectedPanel: null,
+    selectedPanelIds: [],
     disableSaveButton: true,
     disableCancelButton: true,
   };
 
-  selectTest = (testId) => {
-    if (!this.state.selectedTests.includes(testId) && !this.state.defaultTests.includes(testId)) {
+  selectTest = (test) => {
+    const { addDraftLabOrdersToStore, draftLabOrders, deleteDraftLabOrderFromStore } = this.props;
```

Comment:
remove `draftLabOrders` if you ain't using it

Reply:
Done